guides/getting-started/set-up/index.md

```diff
@@ -151,6 +151,12 @@ Before we can initialize the agent, we need to provide the agent with a wallet a
 { type: 'link', label: 'Indy SDK', href: './set-up/indy-sdk', docId: 'getting-started/set-up/indy-sdk/index' }
 ]} />
 
+### Setting up the required ledgers
+
+AFJ is ledger agnostic now, we can select any of the ledger implemntations supported by AFJ
+- [Indy]()
+- [Cheqd](./cheqd)
```

Comment:
You can run AFJ without a ledger if you want. We're trying to keep the set-up as minimal as possible, but list the tutorials page with a specific tutorial in the next steps.

Maybe we can add a tutorial specific to working with cheqd and link that in the next steps? I.e. how to create a cheqd did, etc..? I think for issuance and creating schemas/ credential definition we can keep it general, but we can modify the tutorials to be: "Pick any of the following ledgers you want", and then we add a short tabs section on how to start the tutorial with a specific ledger.

Reply:
There should be a link to the setup instructions somewhere in this doc, https://aries.js.org/guides/getting-started/set-up if you see the website, We can't find setup instructions for any module. 

Maybe a section on configuring modules? what do you think

Reply:
Will be adding the tutorials for cheqd as well, mainly it's for registrar and resolver, other's should be same